index.js

```diff
@@ -1,65 +1,45 @@
 // Import express
 const express = require("express");
 
-// Import file system
-const { writeFile } = require("node:fs/promises");
-
-//Import data from JSON to be read in JavaScript
-const data = require("./data.json");
-
-// Import data from suggestions.json so it can be added to arrayOfSuggestedIdeas
-const arrayOfSuggestedIdeas = require("./suggestions.json");
+const mongoose = require("mongoose");
+
+// Import random activity route
+const randomActivity = require("./routes/randomActivity");
+
+// Import suggested activity route
+const suggestActivity = require("./routes/suggestActivity");
+
+// Connects server to mongodb compass and atlas
+const connectionstring =
```

Comment:
it doesn't connect to Compass. Compass is an app that runs on your desktop, Atlas is the cloud instance of MongoDB
```suggestion
// Connects the API server to MongoDB Atlas
const connectionstring =
```